troubadix/plugins/security_messages.py

```diff
@@ -14,34 +14,69 @@
 #
 #  You should have received a copy of the GNU General Public License
 #  along with this program.  If not, see <http://www.gnu.org/licenses/>.
-import re
 from pathlib import Path
 from typing import Iterator
 
 from troubadix.helper.patterns import ScriptTag, get_script_tag_pattern
 from troubadix.plugin import FileContentPlugin, LinterError, LinterResult
 
+security_message_implementations = [
+    "security_message",
+    "http_check_remote_code",
+    "citrix_xenserver_check_report_is_vulnerable",
+    "citrix_xenserver_report_missing_patch",
+]
+
+
+def _file_contains_security_message(file_content: str) -> bool:
+    """Checks wether a VT content contains a call to security_message
+    or any function known to implement it
+
+    Args:
+        file_content (str): The content of the VT
+    """
+    return any(
+        sec_msg in file_content for sec_msg in security_message_implementations
+    )
```

Comment:
```suggestion
    return any(
        security_message in file_content
        for security_message in SECURITY_MESSAGE_IMPLEMENTATIONS
    )
```
I'd prefer using full variable names rather than abbreviations